src/wrapper-esm.js

```diff
@@ -0,0 +1,35 @@
+/*!
+ * jQuery JavaScript Library v@VERSION
+ * https://jquery.com/
+ *
+ * Copyright OpenJS Foundation and other contributors
+ * Released under the MIT license
+ * https://jquery.org/license
+ *
+ * Date: @DATE
+ */
+// For ECMAScript module environments where a proper `window`
+// is present, execute the factory and get jQuery.
+// For environments that do not have a `window` with a `document`
+// (such as Node.js), expose a factory as module.exports.
+// This accentuates the need for the creation of a real `window`.
+// e.g. var jQuery = require("jquery")(window);
+// See ticket trac-14549 for more info.
+var jQueryOrJQueryFactory = typeof window !== "undefined" && window.document ?
```

Comment:
There's no point in falling back to `this` as we do in the main wrapper. There, it made sense as `this` has some meaning in CommonJS. In ESM, it's just `undefined` at the top level.